R/vector-chr.R

```diff
@@ -82,7 +99,7 @@ string <- function(x, encoding = NULL) {
 #' as_bytes(latin1)
 #' }
 as_utf8_character <- function(x) {
-  enc2utf8(as_character(x))
+  .Call(rlang_unescape_character, as_character(x))
```

Comment:
Sure it's a way of dealing with the problem. My point is that it may have implications beyond fixing this particular problem.